ote/src/clj/ote/tasks/pre_notices.clj

```diff
@@ -91,51 +98,109 @@
                             (when (and no-traffic-routes (> no-traffic-routes 0))
                               (str no-traffic-routes " reitillä tauko liikennöinnissä"))]))]))))
 
-(defn notification-template [pre-notices detected-changes]
-  [:html
-   [:head
-    [:style
-     "table { border-collapse: collapse; font-size: 10px; table-layout: fixed;}
-      table,td,tr,th { border: 1px solid black; }
-      td,tr,th { padding: 5px; }"]]
-   [:body
-    (when (seq pre-notices)
-      [:span
-       [:p [:b "Uudet 60 päivän muutosilmoitukset NAP:ssa"]]
-       [:ul
-        [:li
-         "Muutosilmoitukset on listattu voimaantulopäivämäärän mukaisesti."]
-        [:li
-         "Pääset tarkastelemaan muutosilmoitusta NAP:ssa klikkaamalla reitin nimeä taulukossa."]]
-       (table
-        [{:width "5%" :label "#"}
-         {:width "10%" :label "Reitin nimi"}
-         {:width "10%" :label "Alue"}
-         {:width "10%" :label "Palveluntuottajan nimi"}
-         {:width "20%" :label "Muutoksen tyyppi"}
-         {:width "15%" :label "Muutoksen ensimmäinen voimaantulopäivä"}
-         {:width "30%" :label "Muutoksen tarkemmat tiedot"}]
-        (for [n pre-notices]
-          (pre-notice-row n)))
-       [:br]])
-
-    (when (seq detected-changes)
-      [:span
-       [:p [:b "Uudet tunnistetut liikennöintimuutokset NAP:ssa"]]
-       (table
-        [{:width "20%" :label "Palveluntuottaja"}
-         {:width "20%" :label "Palvelu"}
-         {:width "20%" :label "Alue"}
-         {:width "20%" :label "Aikaa 1. muutokseen"}
-         {:width "20%" :label "Muutokset"}]
-        (detected-change-row detected-changes))
-       [:br]])
-
-    [:p "Tämän viestin lähetti NAP."]
-    [:p "Ongelmia? Ota yhteys NAP-Helpdeskiin," [:br]
-     [:a {:href "mailto:joukkoliikenne@traficom.fi"} "joukkoliikenne@traficom.fi"]
-     [:span " tai 029 534 5454 (arkisin 10-16)"]]]])
+(defn html-divider-border [_]
+  [:div
+
+   [:table {:border "0", :width "80%", :cellpadding "0", :cellspacing "0"}
+    [:tbody
+     [:tr
+      [:td {:style "background:none; border-bottom: 1px solid #d7dfe3; height:1px; width:100%; margin:0px 0px 0px 0px;"} "&nbsp;"]]]]
+   [:br]])
+
+(defn- blue-button [link text]
+  [:table
+   [:tr
+    [:td.btn
+     [:a {:class "mcnButton"
```

Comment:
give the btn class to the a element instead. So the clickable area is as big as the backgroud color.

Reply:
fixed